### PR TITLE
Updated Readme 

### PR DESCRIPTION
**Pull request by gauravgahlot**
Updated [README.md](https://github.com/tinkerbell/boots/blob/master/README.md) to have clear steps for a new user. 

Signed-off-by: Gaurav Gahlot <gaurav.gahlot19@gmail.com>

### CODE REVIEW

**Comment by nathangoulding** [comment]
Also apt, not apt-get and we should provide separate instructions for rpm-based systems.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ This services handles DHCP, PXE, tftp, and iPXE for provisions.
 First, you need to make sure you have [git-lfs](https://github.com/git-lfs/git-lfs/wiki/Installation) installed:
 
 ```
+# install "git-lfs" package for your OS, i.e.:
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+apt install git-lfs
```
Comment (mmlb) [inline README.md:13]: these should be comments too.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ This services handles DHCP, PXE, tftp, and iPXE for provisions.
 First, you need to make sure you have [git-lfs](https://github.com/git-lfs/git-lfs/wiki/Installation) installed:
 
 ```
+# install "git-lfs" package for your OS, i.e.:
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
```
Comment (nathangoulding) [inline README.md:12]: Suggest not silently curl.

**Comment by gauravgahlot** [comment]
> Also apt, not apt-get and we should provide separate instructions for rpm-based systems.

@nathangoulding I will be like duplicating steps from [here](https://github.com/git-lfs/git-lfs/wiki/Installation#rhelcentos). We already have a link to a detailed instruction set.

File: README.md (hunk 2 above)
Comment (vielmetti) [reply README.md]: Strongly suggest not to do `curl | sudo bash` for external dependencies, if any other option is available.

**Pull request merged**